src/raudio.c

```diff
@@ -1139,6 +1139,7 @@ Music LoadMusicStream(const char *fileName)
         }
     }
 #endif
+    else TraceLog(LOG_WARNING, "[%s] Audio fileformat not supported, it can't be loaded", fileName);
```

Comment:
it's ok but it should be:
```c
else TRACELOG(LOG_WARNING, "STREAM: [%s] Fileformat not supported", fileName);
```
to be consistent with similar traces.

Reply:
Thanks, updated.